File: 06.1d_array/4344_over_avg.py
#!/usr/bin/env python

n = int(input())

for _ in range(n):
    xx = [int(x) for x in input().split()]
    num = xx[0]
    scores = xx[1:]
    avg = sum(scores)/num
    num_over_avg = len([x for x in scores if x > avg])
    print("{:1.3f}%".format(num_over_avg/num*100))

File: 06.1d_array/4344_over_avg.cpp
#include <iostream>
#include <vector>
#include <numeric>

using namespace std;

int main() {
    int n, num;
    int avg, num_over_avg;
    scanf("%d", &n);

    for (auto t=0; t<n; ++t) {
        scanf("%d", &num);
        vector<int> v(num);

        for (auto i=0; i<num; ++i)
            scanf("%d", &v[i]);

        avg = accumulate(v.begin(), v.end(), 0)/num;
        num_over_avg = 0;
        for (auto i=0; i<num; ++i)
            if (v[i] > avg) num_over_avg++;

        //cout << double(num_over_avg)/num*100 << endl;
        printf("%1.3f%%\n", double(num_over_avg)/num*100);
    }

    return 0;
}
